### ISSUE TITLE
Include hotstart/ramp options and dependencies (SCHISM-WWM3)?

### ISSUE THREAD

**Issue by VHernaman** [rom-py/rompy-legacy#57]
In our meeting a few weeks ago, we also spoke briefly about including an option/dependency to turn on/off hotstarts in param.nml and wwminput.nml and modify the associated ramp options in param.nml. Typically we use a ramp option for winds, waves, and hydrodynamics, but obviously when using hotstarts you don't want those ramp options turned on. 

But I'm not sure if you wanted this in rompy at this stage, or whether this comes later or is something you want to tackle a different way, but just thought I'd flag it here anyway.

**Comment by tomdurrant** [comment]
Yes, I think we want to include those. If they are just changes in param.nml, that is easy of course, we just have to include them in the template. However, if have to change run times to account for spinnups etc, that is of course more involved. Good one to flag for further discussion
